mono/tests/bug-59281.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Threading;
+
+namespace testlock
+{
+    class Program
+    {
+        public static int _iNbCnxAnnuaire = 2;
```

Comment:
I know this is extracted from the bug report but a little cleanup to make it more readable would be nice.